db/migrate/20221101181505_hide_all_user_tips_for_existent_users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class HideAllUserTipsForExistentUsers < ActiveRecord::Migration[7.0]
+  def up
+    execute "UPDATE user_options SET seen_popups = '{-1}'"
```

Comment:
As I understand it, `-1` means "don't display any user tips, including future ones". Is that expected?

I get skipping user tips about well know features, but we might want to display tips about new features that don't exist at this point.

Reply:
That's fair. Thanks for the suggestion!